src/main/java/com/imsweb/datagenerator/naaccr/NaaccrDataGenerator.java

```diff
@@ -55,10 +55,14 @@
 import com.imsweb.datagenerator.naaccr.rule.tumor.TumorMarkerRule;
 import com.imsweb.datagenerator.naaccr.rule.tumor.TumorRecordNumberRule;
 import com.imsweb.datagenerator.utils.Distribution;
+import com.imsweb.datagenerator.utils.DistributionUtils;
+import com.imsweb.datagenerator.utils.dto.SiteFrequencyDto;
 import com.imsweb.layout.Layout;
 import com.imsweb.layout.LayoutFactory;
 import com.imsweb.layout.record.fixed.naaccr.NaaccrLayout;
 
+import static com.imsweb.datagenerator.utils.DistributionUtils.getAgeGroup;
```

Comment:
Please don't use static import for methods. It makes the code a bit less readable because you don't know in which context (which class) the method exists. We are OK with static import for constants (static final variables) and only if their name is enough to clearly indicate what the variable represents.

Reply:
Fixed.